### ISSUE TITLE
PyTorch version？

### ISSUE THREAD

**Issue by AloveDL** [MICS-Lab/s4_digital_pathology#2]
When PyTorch is in a lower version, there will be changes in torch. fft, which can cause problems during the training process (such as loss =nan).
